main.go

```diff
@@ -58,6 +58,7 @@ func main() {
 	defaultTokenExpiration := flag.Duration("token-expiration", webhooks.DefaultTokenExpirationDefault, "The token expiration")
 	defaultRegion := flag.String("gcp-default-region", "", "If set, CLOUDSDK_COMPUTE_REGION will be set to this value in mutated containers")
 	gCloudImage := flag.String("gcloud-image", webhooks.GcloudImageDefault, "Container image for the init container setting up GCloud SDK")
+	tokenDefaultMode := flag.Int("token-default-mode", webhooks.VolumeModeDefault, "DefaultMode for the token volume")
```

Comment:
I would like to put a caution here.

```suggestion
	tokenDefaultMode := flag.Int("token-default-mode", webhooks.VolumeModeDefault, "DefaultMode for the token volume. CAUTION: if you allow reading from others (e.g. '0444'), the token can read from anyone who can log in to the node.")
```